datawow.gemspec

```diff
@@ -3,8 +3,8 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |s|
   s.name          = 'datawow'
-  s.version       = '0.0.1'
-  s.date          = '2018-08-15'
+  s.version       = '0.0.2'
```

Comment:
I think this changes are pretty much major upgrade,,,consider to adjust the digit to be like `0.1.0` ?

Reply:
Actually must be `1.0.0`.

Reply:
Regard to https://stackoverflow.com/questions/2864448/best-practice-software-versioning